apm-protocol/apm-ui-protocol/src/main/resources/ui-graphql/application-layer.graphqls

```diff
@@ -19,9 +21,6 @@ type ApplicationNode implements Node {
     numOfServerAlarm: Int!
```

Comment:
Adding 'isAlarm' property that represents application alarm.

Reply:
@hanahmily Done.